piksi_tools/ardupilot/mavlink_decode.py

```diff
@@ -8,92 +8,273 @@
 
 Requirements:
 
-  pip install pymavlink
   sudo pip install sbp
 
 """
-from pymavlink.DFReader import DFReader_binary
 from sbp.table import dispatch, _SBP_TABLE
 from sbp.msg import SBP
+from struct import unpack
+from datetime import datetime, timedelta
 
 import json
 
-SBR1_DATASTART = 16
-SBR2_DATASTART = 13
+ARDUPILOT_LOG_HEADER = bytearray([0xA3, 0x95])  # each Ardupilot log frame starts with "0xA3 0x95"
+
+# Timestamp format : 2017-05-26T21:40:15.717000
+TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
+TIME_ORIGIN = "1980-01-06T00:00:00.000000"
+
+FORMAT_SIZE_BYTES = {'B': 1,
+                     'H': 2,
+                     'I': 4,
+                     'Q': 8,
+                     }
+
+"""
+Format characters in the format string for mavlink binary log messages
```

Comment:
can we reference a repo, doc, or source file here for the mavlink format strings?